src/gui/sortablewidgetitems.cpp

```diff
@@ -0,0 +1,129 @@
+/*
+ * Bittorrent Client using Qt4 and libtorrent.
+ * Copyright (C) 2017  Christophe Dumez
```

Comment:
I guess he doesn't wrote this part, put your name here or "The qBittorrent team".